bedrock/products/templates/products/vpn/platforms/desktop.html

```diff
@@ -0,0 +1,173 @@
+
+    {% call vpn_content_well(
+      title='',
+      image_url='img/products/vpn/platforms/placeholder.png',
+      class='vpn-content-well-right-half vpn-content-well-alt-layout'
+    ) %}
+      <p>{{ ftl('vpn-desktop-ability') }}</p>
+      <p>{{ ftl('vpn-desktop-maker-of-firefox') }}</p>
+    {% endcall %}
+
+    {% call vpn_content_well(
+      title=ftl('vpn-shared-what-youll-get'),
+      image_url='img/products/vpn/platforms/placeholder.png',
+      class='vpn-content-well-left-half'
+    ) %}
+      <ul class="mzp-u-list-styled">
+        <li>{{ ftl('vpn-shared-feature-item-protection') }}</li>
+        <li>{{ ftl('vpn-shared-feature-item-access') }}</li>
+        <li>{{ ftl('vpn-shared-feature-item-encryption') }}</li>
+        <li>{{ ftl('vpn-shared-feature-item-bandwidth') }}</li>
+        <li>{{ ftl('vpn-shared-feature-item-logs') }}</li>
+      </ul>
+    {% endcall %}
+
+    {% call vpn_content_well(
+      title=ftl('vpn-desktop-wifi-headline'),
+      image_url='img/products/vpn/platforms/placeholder.png',
+      class='vpn-content-well-right-half'
+    ) %}
+      <p>{{ ftl('vpn-desktop-wifi-copy') }}</p>
+    {% endcall %}
+
+    {% call vpn_content_well(
+      title=ftl('vpn-desktop-privacy-headline'),
+      image_url='img/products/vpn/platforms/placeholder.png',
+      class='vpn-content-well-left-half'
+    ) %}
+      <p>{{ ftl('vpn-desktop-privacy-copy') }}</p>
+    {% endcall %}
+
+    {% call vpn_content_well(
+      title=ftl('vpn-desktop-servers-headline'),
+      image_url='img/products/vpn/platforms/placeholder.png',
+      class='vpn-content-well-right-half'
+    ) %}
+      <p>{{ ftl('vpn-desktop-servers-copy') }}</p>
+    {% endcall %}
+
+    {% call vpn_content_well(
+      title=ftl('vpn-desktop-devices-headline'),
+      image_url='img/products/vpn/platforms/placeholder.png',
+      class='vpn-content-well-left-half'
+    ) %}
+      <p>{{ ftl('vpn-desktop-devices-copy') }}</p>
+    {% endcall %}
+
+    {% call vpn_content_well(
+      title=ftl('vpn-desktop-speed-headline'),
+      image_url='img/products/vpn/platforms/placeholder.png',
+      class='vpn-content-well-right-half'
+    ) %}
+      <p>{{ ftl('vpn-desktop-speed-copy') }}</p>
+    {% endcall %}
+
+    {% call vpn_content_well(
+      title=ftl('vpn-desktop-log-headline'),
+      image_url='img/products/vpn/platforms/placeholder.png',
+      class='vpn-content-well-left-half'
+    ) %}
+      <p>{{ ftl('vpn-desktop-log-copy') }}</p>
+    {% endcall %}
+
+    {% call vpn_content_well(
+      title=ftl('vpn-cta-headline'),
+      image_url='img/products/vpn/platforms/placeholder.png',
+      class='vpn-content-well-right-half'
+    ) %}
+      <a class="vpn-button" href="{{ url('products.vpn.landing') }}" data-link-name="Mozilla VPN">{{ ftl('vpn-shared-cta') }}</a>
+      <div class="vpn-hero-guarantee">
+        <em>{{ ftl('vpn-shared-day-guarantee') }}</em>
+      </div>
+    {% endcall %}
+
+    <!-- {% call vpn_content_well(
```

Comment:
Is this needed? If not can it be removed?

Reply:
Removed.